code/game/machinery/computer/law.dm

```diff
@@ -11,55 +11,57 @@
 	light_range_on = 2
 
 
-	verb/AccessInternals()
-		set category = "Object"
-		set name = "Access Computer's Internals"
-		set src in oview(1)
-		if(get_dist(src, usr) > 1 || usr.restrained() || usr.lying || usr.stat || istype(usr, /mob/living/silicon))
-			return
-
-		opened = !opened
-		if(opened)
-			to_chat(usr, "<span class='notice'>The access panel is now open.</span>")
-		else
-			to_chat(usr, "<span class='notice'>The access panel is now closed.</span>")
+// What the fuck even is this
+/obj/machinery/computer/aiupload/verb/AccessInternals()
```

Comment:
Uh yeah I agree, what the fuck is this? I've never even seen this verb before. And does this mean you can "open" a computer without a screwdriver?

Reply:
Yeah I saw this while doing this and just cried internally